src/main/java/gregtech/api/pattern/BlockPattern.java

```diff
@@ -528,7 +541,7 @@ public BlockInfo[][][] getPreview(int[] repetition) {
                             }
                         }
                         BlockInfo info = infos == null || infos.length == 0 ? BlockInfo.EMPTY : infos[0];
-                        BlockPos pos = setActualRelativeOffset(z, y, x, EnumFacing.NORTH);
+                        BlockPos pos = setActualRelativeOffset(z, y, x, EnumFacing.NORTH, EnumFacing.UP, false);
```

Comment:
Is always using `false` here intentional, or should it use `controllerBase.isFlipped()`?

Reply:
Its intentional, its the JEI preview so it just forces to no rotation and no flip ("default state")